packages/jest-docblock/src/__tests__/index.test.js

```diff
@@ -213,4 +213,182 @@ describe('docblock', () => {
       providesModule: 'apple/banana',
     });
   });
+
+  it('extracts comments from docblock', () => {
+    const code =
+      '/**' +
+      os.EOL +
+      ' * hello world' +
+      os.EOL +
+      ' * @flow yes' +
+      os.EOL +
+      ' */';
+    expect(docblock.parseWithComments(code)).toEqual({
+      comments: 'hello world',
+      pragmas: {flow: 'yes'},
+    });
+  });
+
+  it('extracts multiline comments from docblock', () => {
+    const code =
+      '/**' +
+      os.EOL +
+      ' * hello' +
+      os.EOL +
+      ' * world' +
+      os.EOL +
+      ' * @flow yes' +
+      os.EOL +
+      ' */';
+    expect(docblock.parseWithComments(code)).toEqual({
+      comments: 'hello' + os.EOL + 'world',
+      pragmas: {flow: 'yes'},
+    });
+  });
+
+  it('extracts comments from beginning and end of docblock', () => {
+    const code =
+      '/**' +
+      os.EOL +
+      ' * hello' +
+      os.EOL +
+      ' * @flow yes' +
+      os.EOL +
+      ' * ' +
+      os.EOL +
+      ' * world' +
+      os.EOL +
+      ' */';
+    expect(docblock.parseWithComments(code)).toEqual({
+      comments: 'hello' + os.EOL + os.EOL + 'world',
```

Comment:
With respect to this case of:
```js
/**
 * hello 
 * @flow yes
 *
 * world
 */
```

I'd expect the result of `parseWithComments` to only have a single EOL between the hello and world like:
```js
{ 
  comments: 'hello' + os.EOL + 'world',
  pragmas { flow: 'yes' }
}
```

I might be getting too far into the boonies here, but what would be the current behavior in the case of:
```js
/**
 * @flow
 * @legal 1.0
 * @tomato
 * 
 * hello
 * world
 */
```

Reply:
Current behavior for comments at the end of the docblock worked as expected 👍 .  I was worried it would count more newlines than it should.

Reply:
I modified the test case here to show what I mean: https://github.com/azz/jest/pull/1/files

Reply:
This was actually intended. Because we'll be reprinting comments above pragmas, it made sense to me that:

```js
/**
 * My module does thing special snowflake
 * thing x really well.
 * @flow
 *
 * Copyright (c) 1234 Some Guy
 */
```

To reformat to:

```js
/**
 * My module does thing special snowflake
 * thing x really well.
 *
 * Copyright (c) 1234 Some Guy
 *
 * @flow
 */
```